tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -194,6 +197,33 @@ def __count_total_num_records(self, log_context):
                 level='error')
             exit(1)
 
+    def __assert_sequential_execution(self, log_context):
+        """Assert that no other reparse commands are still executing."""
+        latest_meta_model = ReparseMeta.get_latest()
+        now = timezone.now()
+        is_not_none = latest_meta_model is not None
+        if (is_not_none and not ReparseMeta.assert_all_files_done(latest_meta_model) and
+                not now > latest_meta_model.timeout_at):
```

Comment:
This line is causing a stacktrace on every run, using `-a` or year and quarter params. I'm guessing we have an issue in the conditional right above?
```
vcap@ea6c74e2-d600-4e44-6f68-1c08:~$ python manage.py clean_and_reparse -n -d -a

You have selected to reparse datafiles for FY All and Q1-4. The reparsed files will be stored in new indices and the old indices will be deleted.
These options will delete and reparse ALL (14) datafiles.
Continue [y/n]? y
Traceback (most recent call last):
  File "/home/vcap/app/manage.py", line 31, in <module>
    main()
  File "/home/vcap/app/manage.py", line 27, in main
    execute_from_command_line(sys.argv)
  File "/home/vcap/deps/1/python/lib/python3.10/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/vcap/deps/1/python/lib/python3.10/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/vcap/deps/1/python/lib/python3.10/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/vcap/deps/1/python/lib/python3.10/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/home/vcap/app/tdpservice/search_indexes/management/commands/clean_and_reparse.py", line 302, in handle
    self.__assert_sequential_execution(log_context)
  File "/home/vcap/app/tdpservice/search_indexes/management/commands/clean_and_reparse.py", line 206, in __assert_sequential_execution
    not now > latest_meta_model.timeout_at):
TypeError: '>' not supported between instances of 'datetime.datetime' and 'NoneType'
```

Reply:
It seems like you switched to this branch after testing 3064. If that is the case, and you didnt clean your DB in between, then the `timeout_at` field on the meta model will be `None` because it was not being set until this PR. I can still add a check on the timeout to be safe.

Reply:
Just as an FYSA, the change I added just prevents the exception from happening and does not let reparsing proceed as a safety precaution. When this deploys, you can test that it doesnt throw an exception and then to be able to run you will have to add a timeout value to the field of the latest meta model. Happy to pair up/help get the model in the correct state to allow further testing!